config.ru

```diff
@@ -12,4 +13,8 @@ PuppetWebhook.set :root, File.dirname(__FILE__)
 PuppetWebhook.set :logger, LOGGER
 PuppetWebhook.set :command_prefix, 'umask 0022;'
 
+map '/sidekiq' do
+    run Sidekiq::Web
```

Comment:
tests are failing here due to excessive indentation.

Reply:
fixed